src/main/java/me/drendov/HouseKeeping/HouseKeeping.java

```diff
@@ -67,11 +69,14 @@ public void onEnable() {
         {
             int pluginId = 12268;
             Metrics metrics = new Metrics(this, pluginId);
-        }
-        catch (Throwable ignored){}
+        } catch (Throwable ignored) {}
         PluginDescriptionFile pdfFile = this.getDescription();
         this.version = pdfFile.getVersion();
         logger.info("HouseKeeping v" + this.version + " enabled.");
+
+        // Schedule absence check in 5 seconds
+        ScheduledExecutorService runAbsenceCheck = Executors.newSingleThreadScheduledExecutor();
```

Comment:
Could you please do it configurable, eg if "processAtStart = true", then list players
Default is true.